extensions/2.0/Vendor/MPEG_viewport_recommended/README.md

```diff
@@ -0,0 +1,170 @@
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### MPEG_viewport_recommended.extras
+
+Application-specific data.
+
+* **Type**: [`any`](#reference-any)
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-mpeg_viewport_recommended-viewport"></a>
+## MPEG_viewport_recommended.viewport
+
+specifying a recommended viewport.
+
+**`MPEG_viewport_recommended.viewport` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**name**|`string`|The user-defined name of this object.|No|
+|**translation**|`integer`|Provides a reference to `accessor`|No|
+|**rotation**|`integer`|Provides a reference to `accessor`|No|
+|**type**|`string`|provides the type of camera.|No, default: `"perspective"`|
+|**parameters**|`integer`|Provides a reference to `accessor`|No|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_viewport_recommended.viewport.schema.json](/Extensions/MPEG_viewport_recommended/schema/MPEG_viewport_recommended.viewport.schema.json)
+
+### MPEG_viewport_recommended.viewport.name
+
+The user-defined name of this object.  This is not necessarily unique, e.g., an accessor and a buffer could have the same name, or two accessors could even have the same name.
+
+* **Type**: `string`
+* **Required**: No
+
+### MPEG_viewport_recommended.viewport.translation
+
+Provides a reference to `accessor` where the timed data for the translation of camera object will be made available. The componentType of the referenced accessor shall be `FLOAT` and the type shall be `VEC3`, (x, y, z).
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+### MPEG_viewport_recommended.viewport.rotation
+
+Provides a reference to `accessor` where the timed data for  the rotation of camera object will be made available. The componentType of the referenced accessor shall be `FLOAT` and the type shall be `VEC4`, as a unit quaternion, (x, y, z, w).
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+### MPEG_viewport_recommended.viewport.type
+
+provides the type of camera.
+
+* **Type**: `string`
+* **Required**: No, default: `"perspective"`
+* **Allowed values**:
+    * `"perspective"`
+    * `"orthographic"`
+
+### MPEG_viewport_recommended.viewport.parameters
+
+Provides a reference to `accessor` where the timed data for the perspective or orthographic camera parameters will be made available. The componentType of the referenced accessor shall be `FLOAT` and the type shall be `VEC4`. When the type of the camera object which includes this extension is perspective, `FLOAT_VEC4` means (aspectRatio, yfov,  zfar, znear). When orthographic type, `FLOAT_VEC4` means (xmag, ymag, zfar, znear). The requirements on the camera parameters from ISO/IEC DIS 12113:2021  shall apply.
```

Comment:
"ISO/IEC DIS 12113:2021" -> "ISO/IEC 12113:2021" (also in the schema)

Reply:
implemented.